processing_r/processing/algorithm.py

```diff
@@ -655,6 +656,9 @@ def build_import_commands(self,  # pylint: disable=too-many-locals,too-many-bran
             if isinstance(param, QgsProcessingParameterRasterLayer):
                 rl = self.parameterAsRasterLayer(parameters, param.name(), context)
                 commands.append(self.build_raster_layer_import_command(param.name(), rl))
+            elif isinstance(param, QgsProcessingParameterBand):
+                value = self.parameterAsDouble(parameters, param.name(), context)
```

Comment:
Shouldn't this be parameterAsInt ? I can't see any case where you'd want a non-integer band number here...

Reply:
Oh yes, but in R it is not a problem. However, it is better to change it.

Reply:
Done